v22.2/cdc-transformations.md

```diff
@@ -0,0 +1,281 @@
+
+## Syntax
+
+There are two possible components to CDC transformations:
+
+- _Projections_ will select the columns that you want to emit data from. 
+- _Predicates_ will restrict the selected column change data based on the filters you apply.
+
+~~~ sql
+CREATE CHANGEFEED [INTO sink] [WITH options] AS SELECT projection FROM table [WHERE predicate];
+~~~
+
+Parameter        | Description
+-----------------+---------------
+`sink`             | Specify the sink URL to emit change data to. See [Changefeed Sinks](changefeed-sinks.html) for a list of supported sinks. It is also possible to run a changefeed without a sink `CREATE CHANGEFEED WITH...`, which will send changes to the active SQL session.
+`options`          | Set options on the changefeed. See the [Options](create-changefeed.html#options) table for a full list.
+`projection`       | Select the columns from which to emit data.
+`table`            | Define the table to which the columns belong.
+`predicate`        | Apply optional filters with a `WHERE` clause.
+
+For a SQL diagram of the changefeed expression syntax, see the [`CREATE CHANGEFEED`](create-changefeed.html#synopsis) page.
+
+## Limitations
+
+- It is necessary to pass the [`schema_change_policy='stop'`](create-changefeed.html#schema-policy) option in the changefeed creation statement when using the expression format. [Tracking GitHub issue](https://github.com/cockroachdb/cockroach/issues/85143).
+- You can only apply CDC transformations on a single table in each statement.
+- Some stable functions, notably functions that return MVCC timestamps, are overridden to return the MVCC timestamp of the event.
+- You cannot [alter](alter-changefeed.html) a changefeed that uses an expression.
+
+The following are not permitted in a changefeed expression:
+
+- [Volatile functions](functions-and-operators.html#function-volatility)
+- Sub-select queries
+- [Aggregate](functions-and-operators.html#aggregate-functions) and [window functions](window-functions.html) (i.e., functions operating over many rows).
+
+## Changefeed functions
+
+{{site.data.alerts.callout_info}}
+The following functions are in preview and **subject to change** with continued development.
+{{site.data.alerts.end}}
+
+You can use any function from the [Functions and Operators page](functions-and-operators.html) unless it is marked as ["Volatile"](functions-and-operators.html#function-volatility) or is listed in [Limitations](#limitations). The following functions are changefeed specific:
+
+Function                  | Description
+--------------------------+----------------------
+`cdc_is_delete() `        | Returns `true` if the event is a deletion event.
+`cdc_prev()`              | Returns a JSON representation of a row's previous state. 
+`cdc_updated_timestamp()` | Returns the event's update timestamp. This is typically the MVCC timestamp, but can differ. For example, if the table is undergoing [schema changes](online-schema-changes.html).
+
+## Examples
```

Comment:
I think we should add a geofiltering section. This allows us to create a region-specific changefeed for regional by row tables. It is the same as filtering on a column, but the column is the crdb_region column which is a hidden column that exists for all regional by row tables

Reply:
Oh sure yeah, that makes sense! I will work on adding that.